src/Surfnet/StepupGateway/GatewayBundle/Saml/Proxy/ProxyStateHandler.php

```diff
@@ -19,8 +19,8 @@
 namespace Surfnet\StepupGateway\GatewayBundle\Saml\Proxy;
 
 use Surfnet\StepupGateway\GatewayBundle\Saml\Exception\RuntimeException;
+use Symfony\Component\HttpFoundation\Session\SessionBagInterface;
```

Comment:
SessionBagInterface is not used in this file? Why is an import needed?

If this is really an error, consider tooling that removes unused imports.

Reply:
Thanks, removed that import